<?xml version="1.0" ?>

 <launch>
  <arg name="server" default="128.178.145.104"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883
      frame_id: mocap_world
      broadcast_tf: true
      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node>

  # putting the right lwr in the mocap_world fame
  # the first frame comes from the mocap markers around the robot's base
  # the second frame comes from the launch argument:
  # roslaunch lwr_simple_example real.launch robot_name:=right_lwr robot_world:=right_lwr_base
  <node pkg="tf" type="static_transform_publisher" name="right_lwr_base" args="0 0 0 0 0 0 /right_lwr_markers /right_lwr_base 100"/>

  <node pkg="tf" type="static_transform_publisher" name="left_lwr_base" args="0 0 0 0 0 0 /left_lwr_markers /left_lwr_base 100"/>



  # publish the hardcoded grasp points for the kuka endeffector equipped with the qbhand
  # extracted by running rosrun tf tf_echo guard right_lwr_7_link
  <node pkg="tf" type="static_transform_publisher" name="guard_grasp_right" args="0.631, 0.211, 0.145 0.707, -0.615, -0.234, 0.261 /guard /right_grasp 100"/>

  # extracted by running rosrun tf tf_echo guard left_lwr_7_link
  <node pkg="tf" type="static_transform_publisher" name="guard_grasp_left" args="0.630, -0.299, 0.180 -0.658, 0.679, 0.254, -0.205 /guard /left_grasp 100"/>

  "$(find lwr_simple_example)/launch/rviz/rviz_config.rviz"


  <arg name="rviz_config" default="$(find joint_carry)/config/rviz_config.rviz"/>
    <node pkg="rviz" type="rviz" name="joint_carry_rviz"  respawn="false" args="-d $(arg rviz_config)" output="screen"/>





</launch>